docs/web3-eth-txpool.rst

```diff
@@ -0,0 +1,265 @@
+                gas: "0x44c72",
+                gasPrice: "0x4a817c800",
+                hash: "0xb5b8b853af32226755a65ba0602f7ed0e8be2211516153b75e9ed640a7d359fe",
+                input: "0xb61d27f600000000000000000000000024d407e5a0b506e1cb2fae163100b5de01f5193c00000000000000000000000000000000000000000000000053444835ec580000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                nonce: "0x22",
+                to: "0x7320785200f74861b69c49e4ab32399a71b34f1a",
+                transactionIndex: null,
+                value: "0x0"
+            }]
+            }
+        },
+        queued: {
+            0x976a3fc5d6f7d259ebfb4cc2ae75115475e9867c: {
+                3: [{
+                    blockHash: "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    blockNumber: null,
+                    from: "0x976a3fc5d6f7d259ebfb4cc2ae75115475e9867c",
+                    gas: "0x15f90",
+                    gasPrice: "0x4a817c800",
+                    hash: "0x57b30c59fc39a50e1cba90e3099286dfa5aaf60294a629240b5bbec6e2e66576",
+                    input: "0x",
+                    nonce: "0x3",
+                    to: "0x346fb27de7e7370008f5da379f74dd49f5f2f80f",
+                    transactionIndex: null,
+                    value: "0x1f161421c8e0000"
+                }]
+            },
+            0x9b11bf0459b0c4b2f87f8cebca4cfc26f294b63a: {
+                2: [{
+                    blockHash: "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    blockNumber: null,
+                    from: "0x9b11bf0459b0c4b2f87f8cebca4cfc26f294b63a",
+                    gas: "0x15f90",
+                    gasPrice: "0xba43b7400",
+                    hash: "0x3a3c0698552eec2455ed3190eac3996feccc806970a4a056106deaf6ceb1e5e3",
+                    input: "0x",
+                    nonce: "0x2",
+                    to: "0x24a461f25ee6a318bdef7f33de634a67bb67ac9d",
+                    transactionIndex: null,
+                    value: "0xebec21ee1da40000"
+                }]
+            }
+        }
+    }
+
+------------------------------------------------------------------------------
+
+
+inspect
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.txpool.inspect([, callback])
+
+The property can be queried to list a textual summary of all the transactions currently pending for inclusion in the next block(s), as well as the ones that are being scheduled for future executions. 
+The RPC method used is ``txpool_inspect``.
+
+----------
+Parameters
+----------
+
+
+1. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+
+-------
+Returns
+-------
+
+
+``Object`` - The List of pending and queued transactions summary.
```

Comment:
Please dont just put `object` write the current interface it returns with full details in which field does what else users will just raise issues. Also if its a promise make sure you do `Promise<Type>` so people know that.